android/app/src/main/java/com/dismoi/scout/floating/Manager.kt

```diff
@@ -6,21 +6,18 @@ import android.content.Intent
 import android.content.ServiceConnection
 import android.os.IBinder
 import android.util.Log
-import com.dismoi.scout.floating.layout.Bubble
-import com.dismoi.scout.floating.layout.Layout
-import com.dismoi.scout.floating.layout.Message
 import com.dismoi.scout.floating.FloatingService.FloatingServiceBinder
 
 class Manager private constructor(private val context: Context) {
```

Comment:
A class called a `Manager` don't convey much meaning, this is sooo confusing. But I know you know ;)

Reply:
And I know you know I know !

Reply:
(And actually, you only see deletion here because I’m progressively getting rid of it …)